source/MRViewer/MRRibbonButtonDrawer.h

```diff
@@ -57,6 +57,8 @@ class MRVIEWER_CLASS RibbonButtonDrawer
     /// draw gradient radio button
     MRVIEWER_API static bool GradientRadioButton( const char* label, int* v, int valButton );
 
+    MRVIEWER_API static bool CustomCombo( const char* label, int* v, const std::vector<std::string>& options, bool showPreview = true, const std::vector<std::string>& tooltips = {} );
```

Comment:
mb better have structure like (not to be dependent of vectors size)
```c++
struct ComboOption
{
    std::string caption;
    std::string tooltip;
}
```

Reply:
It doesn't seem suitable, because in many places we will have to reallocate vectors with that data